vignettes/selectorgadget.Rmd

```diff
@@ -86,18 +86,37 @@ Once we've determined the css selector, we can use it in R to extract the values
 
 ```{r}
 library(rvest)
-html <- read_html("http://www.imdb.com/title/tt1490017/")
-cast <- html_nodes(html, "#titleCast .itemprop")
+lego_url <- "http://www.imdb.com/title/tt1490017/"
+html <- read_html(lego_url)
+cast <- html_nodes(html, ".primary_photo+ td a")
 length(cast)
 cast[1:2]
 ```
+Finally, we can extract the text from the selected HTML nodes.
 
-Looking carefully at this output, we see twice as many matches as we expected. That's because we've selected both the table cell and the text inside the cell. We can experiment with selectorgadget to find a better match or look at the html directly.
+```{r}
+html_text(cast, trim = TRUE)
+```
+
+Let's say we're also interested in extracting the links to the actors' pages. We can access html attributes of the selected nodes using `html_attrs()`.   
 
 ```{r}
-cast <- html_nodes(html, "#titleCast span.itemprop")
-length(cast)
+cast_attrs <- html_attrs(cast)
+
+length(cast_attrs)
+cast_attrs[1:2]
+
```

Comment:
Can you please remove this empty line? Otherwise, looks good!

Reply:
Done. Thank you